server/src/main/java/org/elasticsearch/threadpool/ThreadPool.java

```diff
@@ -252,8 +252,8 @@ public long absoluteTimeInMillis() {
         return cachedTimeThread.absoluteTimeInMillis();
     }
 
-    public Counter estimatedTimeInMillisCounter() {
-        return cachedTimeThread.counter;
+    public Supplier<Long> estimatedTimeInMillis() {
```

Comment:
this can be removed and replaced by `ThreadPool::relativeTimeInMillis` on the consumer end